server/api/club/rating/new.post.ts

```diff
@@ -0,0 +1,50 @@
+import { PrismaClient } from '@prisma/client'
+import * as z from 'zod'
+
+const prisma = new PrismaClient()
+
+const requestSchema = z.object({
+  club: z.string(),
+  date: z.string().datetime(),
+  score: z.number().lte(5),
+  comment: z.string(),
+  scope: z.string(),
+})
+
+export default eventHandler(async (event) => {
+  const { auth } = event.context
+
+  if (!auth.userId) {
+    setResponseStatus(event, 403)
+    return
+  }
+
+  const requestBody = await readValidatedBody(event, body => requestSchema.safeParse(body))
+  if (requestBody.success) {
```

Comment:
So you don't need this `if` here.

Reply:
Also, check if `requestSchema.scope` equals to the current scope. If not, return 403.